README.rst

```diff
@@ -18,10 +18,146 @@ HACKING
 
 Find development instructions here: https://github.com/fedora-infra/fmn/#hacking
 
+RUNNING
+-------
+
+There are three components to run in section of the project:
+
+* the fedmsg consumer
+* one or more workers
+* the backend
+
+The consumer is the part of the application that receives the messages from
+fedmsg, it then sends them to the worker(s) who will see whose preferences
+the message match (ie: to whom to send the notification and on which channel).
+The worker(s) then send its results to the backend that receives them and
+just do the IO: sending the email, posting to IRC and so on.
+
+
+To run these parts, simply call, in three different terminals:
```

Comment:
Perhaps add a short note here that people should run rabbitmq?
I see it's further down in the FAQ as well, but just a quick note here might be useful.